### PR TITLE
Remove old members and change some bios

### PR DESCRIPTION
**Pull request by AndrewLester**

### CODE REVIEW

**Comment by AndrewLester** [comment]
Fixes part 3 of #439. We should improve the quality of bios in a separate PR.

File: _pages/team.html [hunk 1]
```diff
@@ -10,210 +10,107 @@
...
+            <p>Anuragi is a captain who specializes in mechanics. She leads teaching of a robotics skills class and organized a Multimedia Learning Center in her home country of Nepal from the summer of 2018 and onward, one of our most ambitious outreach projects. She currently serves as our battery manager. During competitions, she works on our drive team in the role of Technician.</p>
         </div>
         <div>
             <img src="/images/team/andrew-lester.png">
-            <p class="label">🖥️ Programmer 🖥️</p>
+            <p class="label">⭐ Captain ⭐</p>
             <h5>Andrew Lester</h5>
-            <p>Andrew is a programmer with years of experience in Java, Python, and HTML/CSS. He handled our foundational robot code during the 2019 season, and will handle our code in the 2020 season, and has worked on tasks from updating our website to maintaining our open-source projects.</p>
+            <p>Andrew is a captain who specializes in programming. He handled our foundational robot code during the 2019 season, and will handle our code in the 2020 season, and has worked on tasks from updating our website to maintaining our open-source projects.</p>
```
Comment (ErikBoesen) [inline _pages/team.html:22]: Sounds outdated

**Review by ErikBoesen — DISMISSED** [review]

File: _pages/team.html (hunk 1 above)
Comment (ErikBoesen) [reply _pages/team.html:22]: Specifically the "will handle our code in 2020" part

Comment (AndrewLester) [inline _pages/team.html:22]:
```suggestion
            <p>Andrew is a captain who specializes in programming. He handled our foundational robot code during the 2019 and 2020 seasons, and has worked on tasks from updating our website to maintaining our open-source projects.</p>
```

**Review by quirozj — APPROVED** [review]

**Pull request merged**